tweepy/models.py

```diff
@@ -196,17 +196,22 @@ def unfollow(self):
         self.following = False
 
     def lists_memberships(self, *args, **kwargs):
-        return self._api.lists_memberships(user=self.screen_name,
+        return self._api.lists_memberships(user_id=self.id,
+                                           *args,
+                                           **kwargs)
+
+    def lists_ownerships(self, *args, **kwargs):
+        return self._api.lists_ownerships(user_id=self.id,
```

Comment:
This should be part of the separate pull request that adds `API.lists_ownerships`.